docs/devs/e2e-tests.md

```diff
@@ -67,6 +67,37 @@ All configuration variables are set to the desired values by editing and saving
 
 Additionally, the `.env` file contains a range of configuration variables that specify the URLs for accessing the various HIS components in each test environment, as well as the credentials needed to execute the test cases.
 
+### Running tests against a local instance
+
+When developing on Ozone, you will likely run a local instance of the HIS and want to execute the E2E tests against it. To do this, set to `true` for testing on localhost and update the URL coordinates for the **dev** environment profile in the `.env` file.
+
+```
+TEST_LOCALHOST=true
```

Comment:
This brought my attention the actual changes in the E2E project, see the question [here](https://github.com/ozone-his/ozone-e2e/commit/f197b8ae6c2ad83a52d8041bafee91ac22960655#r142967373).

Reply:
I've responded to the question [here](https://github.com/ozone-his/ozone-e2e/commit/f197b8ae6c2ad83a52d8041bafee91ac22960655#r142969789).